### DISCUSSION TITLE
删除线 这样的情景显示不正常

### DISCUSSION THREAD

**Discussion by zhuhaisto** [tamlok/vnote#2182] [Q&A]
![图片](https://user-images.githubusercontent.com/108032980/178101612-2e4883bc-837d-420f-9765-1aef26f890f7.png)

可能我里面有个 - 这样的子项 但是编辑模式正常 预览模式不正常

弱弱的问一句 这种 删除线 可以做快捷键不

**Comment by tamlok** [comment]
其实这个涉及到究竟删除线和无序列表哪个优先级高的问题，有可能阅读模式才是正确的。后面会看一下。

快捷键的话，首先阅读一下快捷键说明，然后编辑用户配置文件，自己添加对应的按键定义即可。

**Reply by zhuhaisto** [reply]
> 其实这个涉及到究竟删除线和无序列表哪个优先级高的问题，有可能阅读模式才是正确的。后面会看一下。
> 
> 快捷键的话，首先阅读一下快捷键说明，然后编辑用户配置文件，自己添加对应的按键定义即可。

这样也不行

https://user-images.githubusercontent.com/108032980/178117332-d386db36-4efa-416b-bdc2-80242e6094e1.mp4
